src/Input/Silk.NET.Input.Common/Interfaces/ICursor.cs

```diff
@@ -33,6 +33,14 @@ public interface ICursor
         /// </remarks>
         CursorMode CursorMode { get; set; }
 
+        /// <summary>
+        /// Determines whether the cursor is locked to the bounds of the view/window of this input context.
+        /// </summary>
+        /// <remarks>
+        /// Only supported by SDL, will have no effect on GLFW.
+        /// </remarks>
+        bool IsConfined { get; set; }
+
```

Comment:
Bit of a weird interface that knows about it's implementors & has a property for only one of them. Should only be a property on SDL implementation, the user can easily check if they are using SDL right now using `is`.

Reply:
The implementations are private by design, so an `is` cast won't work here. Cursor confinement I believe is being implemented in GLFW 3.4 so this will only be temporary, however I see what you're getting at. In a subsequent PR I could add `IsGlfw` or `IsSdl` methods and that'll allow users to conditionally use something like this property, however this PR is big enough as it is.

Reply:
Sounds good!